upgrade httptools to newest version

Please upgrade httptools to the latest version to avoid encountering this error when building httptools locally:
```
      httptools/parser/parser.c:212:12: fatal error: 'longintrepr.h' file not found
        #include "longintrepr.h"
                 ^~~~~~~~~~~~~~~
      1 error generated.
      error: command '/usr/bin/clang' failed with exit code 1
```
because the latest httptools comes with wheel files that provides support for most python3 versions.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -17,7 +17,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.7"
-httptools = "^0.5.0"
+httptools = "^0.6.1"
 
 [tool.poetry.dev-dependencies]
 pytest = "^6.2.4"

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -17,7 +17,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.7"
-httptools = "^0.2.0"
+httptools = "^0.5.0"
 
 [tool.poetry.dev-dependencies]
 pytest = "^6.2.4"

